README.md

```diff
@@ -6,7 +6,7 @@ Ziti represents the next generation of secure, open-source networking for your a
 * [Developer Overview](https://github.com/openziti/ziti/blob/master/doc/001-overview.md)
 * [Local Development](https://github.com/openziti/ziti/blob/master/doc/002-local-dev.md)
 * [Local Deployment](https://github.com/openziti/ziti/blob/master/doc/003-local-deploy.md)
-* [Release Notes](https://github.com/openziti/ziti/blob/master/doc/release-notes.md)
+* [Release Notes](https://github.com/openziti/ziti/blob/master/CHANGLOG.md)
```

Comment:
maybe the title should be CHANGELOG too? i dunno  i still like release-notes.md myself 👍

Reply:
I think titling it Release Notes is ok. I don't have a personal preference between release-notes and CHANGELOG. I think the main driver for change is conforming to the most common name on the principle of least surprise. Having it at the top level in a common name makes it a bit faster for people to find.